Package.swift

```diff
@@ -113,5 +113,6 @@ let package = Package(
 )
 
 var swiftSettings: [SwiftSetting] { [
-//    .enableExperimentalFeature("StrictConcurrency"),
+    .enableExperimentalFeature("StrictConcurrency"),
+    .enableUpcomingFeature("StrictConcurrency")
```

Comment:
I'm enabling both, because otherwise `swift build` and Xcode build disagree about warnings. I seem like they each take one but not the other parameter? I didn't dig into this further since this just works and avoid confusing.

I also tried dropping the settings entirely and setting language mode 6 but that would mean we actually have to fix all the issues (which we can't, because one is in a dependency).